test/test_type_promotion.py

```diff
@@ -402,6 +402,39 @@ def _test_spot(a, b, res_dtype):
                    torch.tensor(1., dtype=torch.complex64, device=device), torch.complex128)
         _test_spot(torch.tensor([1, 1], dtype=torch.bool, device=device), 1., torch.get_default_dtype())
 
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
```

Comment:
@mruberry Is this skip necessary? I thought we always include NumPy now.

Reply:
Correct. We require NumPy as a dependency when running the test suite